src/features/completions.zig

```diff
@@ -1041,6 +1041,11 @@ fn getEnumLiteralContext(
         .l_brace, .comma, .l_paren => {
             dot_context = getSwitchOrStructInitContext(tree, dot_token_index) orelse return null;
         },
+        .r_paren => { // i.e. `(try foo()).`
+            dot_context.likely = .enum_assignment;
+            dot_context.identifier_token_index = token_index - 3;
+            dot_context.need_ret_type = true;
+        },
```

Comment:
```suggestion
```

I strongly disagree, consider https://github.com/zigtools/zls/commit/5d3b02cf65014f84f40907c0fe4bd9ca00f5e8db

Reply:
> I strongly disagree, consider [5d3b02c](https://github.com/zigtools/zls/commit/5d3b02cf65014f84f40907c0fe4bd9ca00f5e8db)

Oh I see, that looks like a much better way of fixing this! :)